doc/extensions/myexampleextension/azext_myexampleextension/__init__.py

```diff
@@ -13,9 +15,21 @@
 def helloworld():
     print('Hello World.')
 
-def load_params(_):
-    pass
 
-def load_commands():
-    from azure.cli.core.commands import cli_command
-    cli_command(__name__, 'hello world', 'azext_myexampleextension#helloworld')
+class HelloWorldCommandsLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        from azure.cli.core.commands import CliCommandType
+        custom_type = CliCommandType(operations_tmpl='azext_myexampleextension#{}')
+        super(HelloWorldCommandsLoader, self).__init__(cli_ctx=cli_ctx,
+                                                       custom_command_type=custom_type)
+
+    def load_command_table(self, args):
+        with self.command_group('hello') as g:
+            g.custom_command('world', 'helloworld')
+        return self.command_table
```

Comment:
For the "simplest" example, they could just register their custom_type to "command_type" in init and then use `g.command`, although this is more idiomatic to how the CLI is structured.

Reply:
I want it to look as similar to CLI to it's easier to convert between the two